admin/core/Background/EncryptAndMigrateBackgroundProcess.php

```diff
@@ -133,15 +132,26 @@ protected function task(mixed $item): bool
      */
     protected function complete(): void
     {
-        if ($this->is_queue_empty()) {
+        if ($this->dbqh->isQueueEmpty($this->identifier)) {
+
             update_option(Option::MIGRATE_FINISH, microtime(true));
 
             $this->dbqh->clearAllUsersPass($this->protocol->getPure());
 
             delete_option(Option::MIGRATE_START);
             delete_option(Option::MIGRATE_FINISH);
             Logger::log(Log::FINISH_MIGRATION);
-            parent::complete();
+
+            update_option(Option::AUTO_MIGRATION, $this->identifier);
+
+            $newUsers = $this->dbqh->getNewUsers();
+
+            if (count($newUsers) > 0) {
+                $this->fh->migrate();
+            } else {
+                update_option(Option::LAST_CHECK, time());
+                parent::complete();
+            }
```

Comment:
Next lines can be removed when functionality will be moved to the background process.